mg/builtins.go

```diff
@@ -31,14 +32,19 @@ func (bcl BultinCmdList) Lookup(name string) (cmd BultinCmd, found bool) {
 	panic("internal error: the `.exec` BuiltinCmd is not defined")
 }
 
+// BuiltinCmds is used to add a list of predefined commands to the state. It
+// implements Reducer interface.
```

Comment:
I'd also not bother mentioning which types implement which interfaces, we should just improve the tools instead.